#CREATE request parameters
string speech_string
--------------------------------
#CREATE response parameters
bool executed

